config/esp32/components/chip/Kconfig

```diff
@@ -678,6 +678,12 @@ menu "CHIP Device Layer"
             help
                 Enable ESP32 Device LocationCapability
 
+        config SEC_CERT_DAC_PROVIDER
+            bool "Use Secure Cert DAC Provider"
+            default n
+            help
+                Enable Secure Cert DAC Provider
```

Comment:
Please provide additional information about from where the certs are read

Reply:
The suggested change is made.